WIP BZ1886229, BZ1937285 - Multipathing, zfcp (#32413)
openshift/openshift-docs: modules/rhcos-enabling-multipath.adoc

Multipathing is only supported when it is activated using the machine config as documented in the following procedure. It must be enabled after {op-system} installation.
====

[IMPORTANT]
====

> To enable multipathing on IBM Z and LinuxONE, additional steps are required during installation. For more information, see "Creating {op-system-first} machines" in _Installing a cluster with z/VM on IBM Z and LinuxONE_.

Reviewer: ```suggestion
On IBM Z and LinuxONE, you can enable multipathing only if you configured your cluster for it during installation.
```

Isn't it too late at this point?
